README.md

```diff
@@ -8,21 +8,63 @@ a set of reusable utilities to allow Wagtail implementors to easily generate dat
 [![PyPI version](https://badge.fury.io/py/streamfield-migration-toolkit.svg)](https://badge.fury.io/py/streamfield-migration-toolkit)
 [![Wagtail Hallo CI](https://github.com/wagtail/streamfield-migration-toolkit/actions/workflows/test.yml/badge.svg)](https://github.com/wagtail/streamfield-migration-toolkit/actions/workflows/test.yml)
 
-## Links
+# Contents
 
-- [Documentation](https://github.com/wagtail/streamfield-migration-toolkit/blob/main/README.md)
-- [Changelog](https://github.com/wagtail/streamfield-migration-toolkit/blob/main/CHANGELOG.md)
-- [Contributing](https://github.com/wagtail/streamfield-migration-toolkit/blob/main/CHANGELOG.md)
-- [Discussions](https://github.com/wagtail/streamfield-migration-toolkit/discussions)
-- [Security](https://github.com/wagtail/streamfield-migration-toolkit/security)
+- [Introduction](#introduction)
+- [Quick Start](#quick-start)
+- [Reference](docs/REFERENCE.md)
+- [Usage](docs/USAGE.md)
+- [Contributing](docs/CONTRIBUTING.md)
 
-## Supported versions
+# Introduction
+
+This package aims to make it easier for developers using StreamField who need to write data 
```

Comment:
A short description of the problem the package addresses would be good here - imagine you're a new Wagtail developer and don't know the internals well yet. In other words, _why_ does a developer need to write data migrations, and what is the cost?